app/pathfinder_svc.py

```diff
@@ -29,8 +31,8 @@ async def import_scan(self, scan_format, report):
             temp_file = '%s/_temp_report_file.tmp' % settings.data_dir
             with open(temp_file, 'wb') as f:
                 f.write(contents)
-            parsed_report = self.parsers[scan_format].parse(temp_file)
-            parsed_report = self.enrich_report(parsed_report)
+            parsed_report = self.parsers[scan_format].parse(temp_file, report)
+            #parsed_report = self.enrich_report(parsed_report)
```

Comment:
stale? or do we actually want to keep this?

Reply:
The report enrichment is feature is still in development, I've made these changes with the NMAP scan and import features in mind, which do not leverage enrichment.  Pathfinder uses a single method for report import/parsing, but we could differentiate this in the future.